keepassxc-browser/content/ui.js

```diff
@@ -134,6 +139,23 @@ kpxcUI.deleteHiddenIcons = function(iconList, attr) {
     }
 };
 
+kpxcUI.isRTL = function(field) {
```

Comment:
`isRTL` is not the best name. input text could be RTL and be either left *or* right aligned.
Icon position simply needs to know if the effective text is aligned right or left - than position the icon
on the opposite side. So perhaps a better name would be `isRightAligned` or some such?